### DISCUSSION TITLE
Welcome to Solaris Discussions!

### DISCUSSION THREAD

**Discussion by icarusinthewings** [NovaoTech/solaris#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by duubi-official** [comment]
![Polaris Cover](https://user-images.githubusercontent.com/87050344/125089918-650c0400-e094-11eb-8958-ee65b54baf15.png)
I made a banner! Tell me what you think!

**Reply by icarusinthewings** [reply]
Oooh perfect. Can I  get the sparkle independent of the project as an SVG?

**Reply by duubi-official** [reply]
Okay! It will just take me a sec...
